jwst/datamodels/schemas/ifuimage.schema.yaml

```diff
@@ -0,0 +1,41 @@
+allOf:
+- $ref: core.schema.yaml
+- $ref: bunit.schema.yaml
+- type: object
+  properties:
+    data:
+      title: The science data
+      fits_hdu: SCI
+      default: 0.0
+      ndim: 2
+      datatype: float32
+    dq:
+      title: Data quality array
+      fits_hdu: DQ
+      default: 0
+      datatype: uint32
+    err:
+      title: Error array
+      fits_hdu: ERR
+      default: 0.0
+      datatype: float32
+    zeroframe:
+      title: Zeroframe array
+      fits_hdu: ZEROFRAME
+      default: 0.0
```

Comment:
Agreed and done for som eof them. The pathloss correction schema does not have defaults and yet these are the ones that show up. I wonder if it's a bug in models for 1D arrays.